packages/url_launcher/url_launcher/example/ios/RunnerUITests/URLLauncherUITests.m

```diff
@@ -27,19 +27,13 @@ - (void)testLaunch {
   ];
   for (NSString* buttonName in buttonNames) {
     XCUIElement* button = app.buttons[buttonName];
-    if (![button waitForExistenceWithTimeout:30.0]) {
```

Comment:
I had copied this pattern from some other existing test, but I think just the `XCTAssertTrue` is less busy.